icubam/map_builder.py

```diff
@@ -76,7 +76,7 @@ def prepare_jsons(
       'lat': center_icu.lat,
       'lng': center_icu.long
     } if center_icu else None
-    for covid in [True, False]:
+    for covid in [True, False, None]:
```

Comment:
It would help to have a comment (here or in another file) explaining what's is covid=None.